content/en/blog/2022/etcd-integrates-continuous-fuzzing.md

```diff
@@ -0,0 +1,31 @@
+---
+title: etcd Integrates Continuous Fuzzing
+spelling: cSpell:ignore Gyuho
```

Comment:
I don't think we need  `spelling:` but it's okay, will remove it later if needed.